goth/address.py

```diff
@@ -67,7 +67,7 @@ def _with_default(self, mapping: Mapping[str, object]):
 
 ROUTER_HOST = "router"
 ROUTER_PORT = 7477
-ROUTER_PROTOCOL = "tcp"
+ROUTER_PROTOCOL = "udp"
```

Comment:
Is this / should this be backward-compatible with `ya-sb-router`?

Reply:
its not used :( we should remove those variables for now...

reverted the change to keep it backwards compatible and cleaner